tests/contract/test_log_emission.py

```diff
@@ -36,7 +32,7 @@ def test_log_emission(smc_handler):  # noqa: F811
     # Check that log was successfully emitted
     log = shard_tracker.get_register_notary_logs()[0]
     assert getattr(log, 'index_in_notary_pool') == 0 and \
```

Comment:
Not part of the PR, but why do we use `getattr` instead of just `log.index_in_notary_pool` here?